src/main/fc/settings.yaml

```diff
@@ -136,7 +136,7 @@ tables:
     values: ["HIGH", "MEDIUM", "LOW"]
     enum: dynamicFilterRange_e
   - name: pidTypeTable
-    values: ["NONE", "PID", "PIFF", "AUTO"]
+    values: ["NONE", "PID", "PIDFF", "AUTO"]
```

Comment:
To be honest, I would not change the name. That would mess up the backup/restore for many users and is not that crusial

Reply:
That's a fair point. I thought it would be good to be consistent, especially if in the future other controllers will be added (who knows?).